### DISCUSSION TITLE
Spigot download issue with Bukkit server

### DISCUSSION THREAD

**Discussion by Clownsonged** [itzg/docker-minecraft-server#2722] [General]
Restarted my Bukkit container, ran into this issue and attempted to rebuild the container and am getting the error below. When I visit https://getbukkit.org/download/spigo I also am met with a "Bukkit currently has no sponsors for you" message. Is this an issue that I can trace with my container etc or is it an issue out of my control? Any advice appreciated.

Docker compose:
version: "3"
services:
  minecraft:
    image: itzg/minecraft-server
    volumes:
      - minecraftdata:/data
      - /home/Minecraft/plugins:/plugins
    ports:
      - 25565:25565
    environment:
      EULA: "TRUE"
      TYPE: "BUKKIT"
      OPS: "Clownsong"
      ENABLE_WHITELIST: "TRUE"
      ENFORCE_WHITELIST: "TRUE"
    tty: true
    stdin_open: true
    restart: unless-stopped
volumes:
  minecraftdata:
    external: true

Error in log:
[init] Resolving type given BUKKIT
2024/03/16 18:17:04 Unable to find an element with attribute matcher property=og:title
[init] ERROR: failed to retrieve latest version from https://getbukkit.org/download/spigot -- site might be down

**Comment by itzg** [comment]
Wow, they lost their domain

![image](https://github.com/itzg/docker-minecraft-server/assets/988985/ba7d0d29-97e2-410d-bd6d-9a6e042cdead)

I recommend you switch to Paper.

**Reply by Clownsonged** [reply]
Oh, amazing, if I switch over to paper and keep the same volume for minecraft data will it all resolve itself? Or will I need to manually port over the world etc to a new volume created for the paper compose? Apologies if this is a simple question

**Reply by itzg** [reply]
I believe it is fully compatible, so only the `TYPE` needs to be changed.

**Reply by Clownsonged** [reply]
Amazing, thankyou for the speedy replies and hard work, it is very much appreciated <3
